server.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"budget-helper/database"
+	"budget-helper/router"
+	"budget-helper/users"
+	"log"
+	"net/http"
+)
+
+type Server struct {
+	Router *router.Router
+	Port   string
+}
+
+func NewServer(port string, db *database.Database) *Server {
+	userRepo := users.NewUserRepo(db)
+
+	router := router.NewRouter(userRepo)
+	router.RegisterRoutes()
+
+	// TODO: Check that port is a valid numerical string
```

Comment:
It's fine, program will crash immediately if we don't have a valid port. In general, hard crashes that happen instantly on startup are fine to let crash, as you will notice them immediately.